autobotAI_integrations/integrations/nessus/__init__.py

```
@@ -107,33 +217,32 @@ def get_forms():
             ],
         }
 
-    @staticmethod
-    def get_all_python_sdk_clients():
+    @classmethod
+    def get_all_python_sdk_clients(cls, integration_type=None):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `get_all_python_sdk_clients`

(ANN206)